packages/core/admin/ee/server/services/review-workflows/workflows/index.js

```diff
@@ -19,8 +27,21 @@ const processFilters = ({ strapi }, filters = {}) => {
 
 module.exports = ({ strapi }) => {
   const workflowsContentTypes = workflowsContentTypesFactory({ strapi });
+  const limits = {
+    workflows: MAX_WORKFLOWS,
+    stagesPerWorkflow: MAX_STAGES_PER_WORKFLOW,
+  };
 
   return {
+    register({ workflows, stagesPerWorkflow }) {
+      if (!Object.isFrozen(limits)) {
+        limits.workflows = clampMaxWorkflows(workflows || limits.workflows);
```

Comment:
I don't think it makes sense to have 0 workflows as the limit, as it would lock the feature with our default workflow.
If so, we might "just" want to disable review workflow 🤔